ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json

```diff
@@ -15,10 +92,222 @@
         {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 5},
         {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0}
       ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 4},"window": {"start": 0, "end": 2, "type": "time"},"timestamp": 1},
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 1},"window": {"start": 2, "end": 4, "type": "time"},"timestamp": 2}
+      ]
+    },
+    {
+      "name": "should handle null values",
+      "statements": [
+        "CREATE STREAM INPUT (ID STRING KEY, COL1 STRING) WITH (kafka_topic='input_topic',value_format='JSON');",
+        "CREATE TABLE OUTPUT AS SELECT ID, COUNT(COL1) as COUNT FROM INPUT WINDOW TUMBLING (SIZE 2 MILLISECONDS, GRACE PERIOD 1 MILLISECONDS) GROUP BY ID EMIT FINAL;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": null},"timestamp": 0},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": null},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 2},
+        {"topic": "input_topic", "key": null, "value": {"COL1": "v1"},"timestamp": 1},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": null},
+        {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 5}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 2},"window": {"start": 0, "end": 2, "type": "time"},"timestamp": 1},
```

Comment:
how do we get a count of 2 here? I might be missing something, but I only see one value that should match (or is `null` timestamp treated as `0`?):
```
         {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 1}, 
```

Reply:
Yeah the null timestamp is treated as 0 for some reason

Reply:
cc @vvcephei - can you confirm? though I suppose in reality you'd never have a 0 timestamp so this window would always be past the grace period and it's only a testing "bug"

Reply:
You probably need to just take a look at the behavior of the thing that reads these test specs. Timestamp is a primitive `long` in Streams (and Kafka in general), so there's nothing that would automatically convert a `null` to `0`. If anything, you might get a NullPointerException trying to cast `null` to `long` (I.e., unboxing `Long`) at some point.